documents/affiliated_package_review_process.md

```diff
@@ -0,0 +1,172 @@
+ # Procedure for the proposal, review, and acceptance of Astropy-affiliated packages
```

Comment:
Is it "astropy-affiliated" or "astropy affiliated"?  I've always used the latter, but I guess there's an argument for other.  Regardless, consistency is probably good.

Reply:
APE15 does have the hyphen so maybe that's the answer?